plugins/pulp_rpm/plugins/importers/yum/repomd/updateinfo.py

```diff
@@ -62,7 +66,10 @@ def process_package_element(element):
         for package in collection['packages']:
             if package.get('reboot_suggested') is not None:
                 package_info['reboot_suggested'] = True
-                break
+            if package.get('relogin_suggested') is not None:
+                package_info['relogin_suggested'] = True
```

Comment:
Is here an assumption that if tag is present in the package description then it always has a `True` value? Is it intentional? Can tag be present but has a `False` value?

Reply:
I assumed this worked in the same way as reboot_suggested, but it's a good idea to revise this to work with both `True` and `False` possible values